applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -531,7 +531,7 @@ Every Bluetooth LE bridged device uses a separate Bluetooth LE connection, so yo
 Since the Matter stack uses one Bluetooth LE connection for commissioning, the maximum number of connections you can use for bridged devices is one less than is configured using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option.
 
 Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
-The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
+The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10`` (which effectively means ``9`` bridged devices).
```

Comment:
```suggestion
The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``, which effectively means nine bridged devices.
```
The number of devices isn't an input value in this case.

Reply:
Sorry, but I'm sort of confused :) Why should we write some numbers as a digits and the other ones using words? And also why should we write nine, but in other examples 19?

Reply:
Ah, that's a general rule the docs team uses: numbers zero to nine are written as the word, unless it's a measurement, input value, or is compared to a number that isn't written out (for example, "select 4 of the 18 available options").

Though I guess in this case, the 9 is in relation to the input value 10, so it could be left as the numeral instead of written out. So yeah, feel free to leave the 9 as the number.

Reply:
I see, thanks for clarification!